### ISSUE TITLE
Zoroark causing crashes

### ISSUE THREAD

**Issue by dramamine** [dramamine/leftovers-again#4]
Running these:

```
npm run start:quick -- tiago-duarte --scrappy --matches=1
npm run start:quick -- stabby
```

I ended up getting this message

```
             |                  Electrode            | XXXOOO | OOXX?? |            Tentacruel                 | suckerpunch 
BEST POKEMON SELECTED AGAINST OPPONENT!
Found zoroark! replacing him!
TypeError: Cannot read property 'replace' of undefined
    at Pokemon.useCondition (/mnt/PERPETUAL_GAZE/leftovers-again/lib/model/pokemon.js:390:28)
    at Pokebarn.replace (/mnt/PERPETUAL_GAZE/leftovers-again/lib/model/pokebarn.js:89:15)
    at BattleStore.handleReplace (/mnt/PERPETUAL_GAZE/leftovers-again/lib/model/battlestore.js:210:17)
    at BattleStore.handle (/mnt/PERPETUAL_GAZE/leftovers-again/lib/model/battlestore.js:107:29)
    at Battle.handle (/mnt/PERPETUAL_GAZE/leftovers-again/lib/battle.js:121:18)
    at Listener.relay (/mnt/PERPETUAL_GAZE/leftovers-again/lib/listener.js:48:43)
    at WebSocket.handleMessage (/mnt/PERPETUAL_GAZE/leftovers-again/lib/connection.js:72:32)
    at emitTwo (events.js:106:13)
    at WebSocket.emit (events.js:191:7)
    at Receiver.ontext (/mnt/PERPETUAL_GAZE/leftovers-again/node_modules/ws/lib/WebSocket.js:841:10)
```

Other bot saw some Zoroark stuff too:

```
             |                   Primeape            | XXOOOO | OXX??? |            Espeon                     | dazzlingglea
             |                   Primeape            | XXOOOO | OXX??? |            Espeon                     | dazzlingglea
OK, found my zoroark.
             |                 Tentacruel            | XXOOOO | OXX??? |            Espeon                     | dazzlingglea
OK, found my zoroark.
 suckerpunch |                 Tentacruel            | XXOOOO | OXX??? |            Espeon                     | morningsun  
OK, found my zoroark.
 suckerpunch |                 Tentacruel            | XXOOOO | OXXX?? |            Electrode                  |             
Found zoroark! replacing him!
TypeError: Cannot read property 'replace' of undefined
```

**Comment by dramamine** [comment]
Believe this got fixed in 0.9.3, need to verify

**Comment by dramamine** [comment]
This got fixed for real in 0.9.6
